src/vs/workbench/contrib/chat/browser/chatEditing/chatEditingModifiedDocumentEntry.ts

```diff
@@ -271,7 +271,7 @@ export class ChatEditingModifiedDocumentEntry extends AbstractChatEditingModifie
 		}
 	}
 
-	acceptAgentEdits(textEdits: TextEdit[], isLastEdits: boolean, responseModel: IChatResponseModel): void {
+	async acceptAgentEdits(_resource: URI, textEdits: TextEdit[], isLastEdits: boolean, responseModel: IChatResponseModel): Promise<void> {
```

Comment:
Move to base class, as notebooks need to apply the edits to the cells.
Also the text Model associated with the cell may not be available and might have to be created using the `ITextModelService` which in turn is async.